pkg/crc/oc/clusteroperator.go

```diff
@@ -25,19 +25,27 @@ type K8sResource struct {
 	} `json:"items"`
 }
 
-func GetClusterOperatorStatus(oc OcConfig) (bool, error) {
-	allAvailable := true
+type ClusterStatus struct {
+	Available  bool
+	Degraded   bool
+	Processing bool
```

Comment:
`Processing` or `Progressing`?

Reply:
yes it is `Progressing`, Good catch :+1: